DESCRIPTION

```diff
@@ -37,6 +38,8 @@ Suggests:
 VignetteBuilder:
     knitr
 Remotes:
+    jeroen/curl,
```

Comment:
Want to drop this now?

Reply:
Yes, we can use CRAN curl - we got our change in in time! Amended in [51144e4](https://github.com/tidyverse/elmer/pull/158/commits/51144e4662ed7c69e471e28b7c8f3653c99ed1e4).